index.html

```diff
@@ -138,11 +138,11 @@ <h3 id="form-header">Get Started</h3>
           method="POST"
           >
           <label for="name"></label>
-          <input class="st-input" type="text" name="name" placeholder="Your name" maxlength="30" required>
+          <input class="st-input" type="text" name="name" placeholder="Your name" required maxlength="30">
           <label for="email"></label>
           <input class="st-input" type="email" name="email" placeholder="Email address" required>

```

Comment:
Good job with the validations 👍 But let's fix the font-sizes like on the Figma template. Currently, it is too small to read.
<img width="475" alt="Screenshot 2023-01-24 at 8 13 29 PM" src="https://user-images.githubusercontent.com/25887253/214364613-c1f00cf0-8af0-4957-823f-06ee55c834c0.png">
<img width="801" alt="Screenshot 2023-01-24 at 8 14 01 PM" src="https://user-images.githubusercontent.com/25887253/214364616-7b3a639c-a034-48a2-8039-6dabdff19b0e.png">

Reply:
Done. But not the same font size settings. I have already given my opinion on the top comment.